dev/preview_docs.py

```diff
@@ -74,16 +93,16 @@ def main():
     comment_body = f"""
 {marker}
 
-### Documentation preview will be available [here]({artifact_url}).
+### Documentation preview for {args.commit_sha} will be available [here]({artifact_url}).
```

Comment:
Included a commit ID in the message to make it easier to find which commit the preview corresponds to.